src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3048,6 +3050,50 @@ export class ResultsViewPageStore {
         []
     );
 
+    readonly facetsClinicalDataForSamples = remoteData<ClinicalData[]>(
+        {
+            await: () => [this.studies, this.samples],
+            invoke: () =>
+                this.getClinicalData(
+                    'SAMPLE',
+                    this.studies.result!,
+                    this.samples.result,
+                    ['FACETS_WGD', 'FACETS_PURITY']
```

Comment:
These strings need to be put into constants. They are reused.

Reply:
where do you recommend such constants be moved?

Reply:
@ao508 if these constants will not be shared, probably a file under the feature. Otherwise, you could use constants.ts under the shared folder

Reply:
@zhx828 there are many other such clinical attribute identifiers hardcoded into this file. I will extract those from here as well and relocate to constants.ts

Reply:
@sheridancbio there are some of the places in the portal we haven't standardize this 
practice yet, so thanks for doing this. Just make sure if the string is reused in few places, then worth to move to constants. If not, it's up to you whether you wanna move.